src/AppInstallerCLICore/ExecutionReporter.cpp

```diff
@@ -100,6 +101,61 @@ namespace AppInstaller::CLI::Execution
         }
     }
 
+    bool Reporter::PromptForBoolResponse(Resource::LocString message, Level level)
+    {
+        const std::vector<PromptOption> options
+        {
+            PromptOption{ Resource::String::PromptOptionYes, true },
+            PromptOption{ Resource::String::PromptOptionNo, false },
+        };
+
+        auto out = GetOutputStream(level);
+        out << message << std::endl;
+
+        // Try prompting until we get a recognized option
+        for (;;)
+        {
+            // Output all options
+            for (size_t i = 0; i < options.size(); ++i)
+            {
+                out << PromptEmphasis << "[" + options[i].Hotkey + "] " + options[i].Label;

```

Comment:
>options[i].Hotkey [](http://example.com/codeflow?start=47&length=17)

would the hot key be a problem in other languages? I'd think hardcoded y, n as hot keys